Libraries/Opc.Ua.Client/Subscription.cs

```diff
@@ -2241,7 +2246,7 @@ private async Task OnMessageReceivedAsync(CancellationToken ct)
                         {
                             // tolerate if a single request was received out of order
                             if (ii.Next.Next != null &&
-                                ii.Value.Timestamp.AddMilliseconds(kRepublishMessageTimeout) < DateTime.UtcNow)
+                                ii.Value.TimestampMonotonic + kRepublishMessageTimeout < HiResClock.TickCount)
```

Comment:
this doesn't work in a rollover?
```suggestion
                                (HiResClock.TickCount - ii.Value.TimestampMonotonic) < kRepublishMessageTimeout)
```

Reply:
fixed